collect_app/src/main/res/layout/automatic_project_creator_dialog_layout.xml

```diff
@@ -0,0 +1,104 @@
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
+
+        <com.google.android.material.appbar.MaterialToolbar
+            android:id="@+id/toolbar"
+            style="@style/Widget.AndroidShared.Toolbar.Primary.CloseIcon"
+            android:layout_width="match_parent"
+            android:layout_height="?attr/actionBarSize" />
+
+    </com.google.android.material.appbar.AppBarLayout>
+
+    <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        app:layout_constraintBottom_toBottomOf="@+id/shadow_up"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/add_new_project_toolbar">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:paddingBottom="@dimen/margin_standard"
+            app:layout_constraintBottom_toTopOf="@+id/constraintLayout">
+
+            <com.google.android.material.textview.MaterialTextView
+                android:id="@+id/title"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/scan_configuration_qr_code"
+                android:layout_marginTop="@dimen/margin_standard"
+                style="@style/TextAppearance.Collect.Subtitle1"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent" />
+
+            <com.journeyapps.barcodescanner.DecoratedBarcodeView
+                android:id="@+id/barcode_view"
+                app:layout_constraintWidth_percent="0.75"
+                app:layout_constraintDimensionRatio="w,1:1"
+                android:layout_width="0dp"
+                android:layout_height="0dp"
+                android:layout_marginTop="@dimen/margin_extra_large"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/title" />
+
+            <com.google.android.material.textview.MaterialTextView
+                android:id="@+id/or"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/or"
+                android:layout_marginTop="@dimen/margin_standard"
+                style="@style/TextAppearance.Collect.Subtitle1"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/barcode_view" />
+
+            <com.google.android.material.button.MaterialButton
+                android:id="@+id/configure_manually_button"
```

Comment:
Could we introduce a [theme overlay](https://medium.com/androiddevelopers/android-styling-themes-overlay-1ffd57745207) for `OutlinedButton` (like the ones in `material_theme_polyfill`) to solve the color problems here? When we switch to a MaterialComponents theme/proper colors it will be more awkward if we have color customizations on individual views.

Reply:
Actually that color is not needed and I removed it.